# Frame de referencia
string frame_id
# rotate == 0 ? mirar al punto definido por vector. Else, rotar yaw pitch seg√∫n vector.x vector.y
int32 rotate
# datos
geometry_msgs/Vector3 vector
